src/app/components/client/ExposureCard.tsx

```diff
@@ -151,15 +180,7 @@ const ScanResultCard = (props: ScanResultCardProps) => {
               {
                 // TODO: Add unit test when changing this code:
                 /* c8 ignore next 7 */
-                exposureImg ? (
-                  <Image
-                    className={styles.exposureImage}
-                    alt=""
-                    src={exposureImg}
-                  />
-                ) : (
-                  <FallbackLogo name={scanResult.data_broker} />
-                )
+                exposureImg ?? <FallbackLogo name={scanResult.data_broker} />
```

Comment:
Not sure this matters, but I noticed we used a `cleanedName` version above.
```js
const cleanedName = exposureData.data_broker.split(".")[0];
...
setDataBrokerImage(<FallbackLogo name={cleanedName} />);
```

Although maybe `cleanedName` is irrelevant if we're only looking at the first character anyways and this comment is delightfully moot.
```js
props.name.substring(0, 1)
```